### PR TITLE
MACT-7: Implement Credit Balance + Features step

### PR DESCRIPTION
**Pull request by guillegr123**

### CODE REVIEW

File: submodules/wizard/wizard.js [hunk 1]
```diff
@@ -861,24 +878,204 @@ define(function(require) {
...
+								var flattenedFeatures = _.concat([ feature.name ], featureTreeToList(feature.features));
+								return flattenedFeatures;
+							});
+						},
+						controlCenterFeatureList = _
+							.chain(controlCenterFeatureTree)
+							.flatMap(function(category, index) {
+								return _.map(category.features, function(feature) {
+									return _.merge({
+										categoryIndex: index	// TODO: Calculate top position based on category index and feature index (42px for category 0, 50px for the rest, and 40px per feature index)
```
Comment (azefiel) [inline submodules/wizard/wizard.js]: What about this TODO?

File: submodules/wizard/wizard.js [hunk 2]
```diff
@@ -1007,6 +1204,34 @@ define(function(require) {
 			}
 		},
 
+		/**
+		 * Gets the currency symbol for the current configuration
+		 * Based on: monster.util.formatPrice()
+		 */
+		wizardGetCurrencySymbol: function() {
```
[the remark this answers is not in the document]
Comment (azefiel) [reply submodules/wizard/wizard.js]: [Here](https://github.com/2600hz/monster-ui/blob/master/docs/monster-util/getCurrencySymbol().md) ya go!

[the remark this answers is not in the document]
Comment (guillegr123) [reply submodules/wizard/wizard.js]: I completely missed that function, thanks!

File: submodules/wizard/wizard.js (hunk 1 above)
Comment (guillegr123) [reply submodules/wizard/wizard.js]: I forgot about it. At the end it was not necessary, so I cleaned up that block.

**Pull request merged**